packages/mrtd/src/DidCommMrtdModule.ts

```diff
@@ -53,4 +53,27 @@ export class DidCommMrtdModule implements Module {
       }),
     )
   }
+
+  /**
+   * Initializes the module (e.g. by pre-loading the CSCA Master List if configured).
+   */
+  public async initialize(agentContext: AgentContext): Promise<void> {
+    const logger = agentContext.dependencyManager.resolve<Logger>(InjectionSymbols.Logger)
```

Comment:
Since we are planning to remove InjectionSymbols in future versions of credo-ts, maybe it would be better if you get it from the config directly:

```suggestion
    const logger = agentContext.config.logger
```